mmpose/apis/webcam/utils/pose.py

```diff
@@ -3,8 +3,8 @@
 
 
 def get_eye_keypoint_ids(dataset_meta: Dict) -> Tuple[int, int]:
-    """A helpfer function to get the keypoint indices of left and right eyes
-    from the dataset meta information.
+    """A help function to get the keypoint indices of left and right eyes from
```

Comment:
I think the term 'helper function' is ok. Is there concern about it?

Reply:
Actually not. I found the typo 'helpfer' and simply replaced it with 'help'. I'll change it now. 

BTW, maybe we also need to fix this type in master?